src/utils/log.ts

```diff
@@ -9,21 +9,24 @@ export interface Logger {
   fatal: (message: any) => void;
 }
 
+export const SUPPORTED_LOG_CONFIG_LEVELS = <const>[
+  "debug",
+  "info",
+  "warn",
+  "error",
+  "fatal",
+  "none",
+];
+
+export type LogConfigLevel = (typeof SUPPORTED_LOG_CONFIG_LEVELS)[number];
+
```

Comment:
We don't need the `SUPPORTED` prefix because there are no unsupported levels.

```suggestion
export const LOG_CONFIG_LEVELS = <const>[
  "debug",
  "info",
  "warn",
  "error",
  "fatal",
  "none",
];

export type LogConfigLevel = (typeof LOG_CONFIG_LEVELS)[number];

```